ansible_collections/arista/avd/roles/eos_designs/python_modules/network_services/utils.py

```diff
@@ -333,3 +333,8 @@ def _virtual_router_mac_address(self) -> str | None:
     @cached_property
     def _underlay_filter_redistribute_connected(self) -> bool:
         return get(self._hostvars, "underlay_filter_redistribute_connected", default=True) is True
+
+    @cached_property
+    def _invalid_characters(self) -> list:
```

Comment:
You could also link to some canonical documentation about what chars are invalid, if Arista has it handy.